concepticondata/conceptlists/Yao-2017-1100.tsv

```diff
@@ -0,0 +1,1101 @@
+Yao-2017-1100-830	830	手铐	cuff			4	4.27	4.27	2.77	3.68	3.97
+Yao-2017-1100-831	831	手枪	rifle	GUN	1566	2.8	6.49	4.66	4.39	6.08	5.78
+Yao-2017-1100-832	832	手艺	skill			7.04	6.18	6.75	6.44	7.05	6.98
+Yao-2017-1100-833	833	守灵	vigil			4.86	5.86	4.47	4.25	4.84	5.65
+Yao-2017-1100-834	834	守则	rules			1.86	6.91	5.66	5.22	6.54	6.05
+Yao-2017-1100-835	835	首创	innovate			5.97	4.95	5.54	5.15	7.37	6.58
+Yao-2017-1100-836	836	首饰	jewelry			4.86	6.04	6.04	5.96	7.37	6.6
+Yao-2017-1100-837	837	寿命	life span			5.44	3.44	5.32	5.09	5.37	4.76
+Yao-2017-1100-838	838	寿桃	peaches offered as a birthday present			6.05	6.09	5.77	5.73	4.29	4.66
+Yao-2017-1100-839	839	寿星	the long-living people			5.61	4.66	5.37	5.09	3.59	4.86
+Yao-2017-1100-840	840	枢纽	hub			6.66	7.34	6.83	6.36	7.67	7.47
+Yao-2017-1100-841	841	淑女	lady	LADY	2471	6.17	6.19	7.06	5.58	7.28	6.6
+Yao-2017-1100-842	842	舒畅	comfortable			4.03	3.43	4.03	6.73	5.48	4.84
+Yao-2017-1100-843	843	蔬菜	vegetables	VEGETABLES	924	6.64	3.89	7.53	7.78	7.49	6.7
+Yao-2017-1100-844	844	熟悉	familiar	FAMILIAR	3005	5.36	3.95	6.09	5.45	4.37	4.5
+Yao-2017-1100-845	845	暑假	summer holiday			7.58	5.25	4.58	6.42	5.51	5.91
+Yao-2017-1100-846	846	属性	nature			5.34	4.71	4.83	5.22	5.17	5.77
+Yao-2017-1100-847	847	树冠	tree crown			3.68	6.14	4.36	4.59	3.76	5.05
+Yao-2017-1100-848	848	树林	woods	FOREST	420	6.2	4.57	5	5.33	4.93	4.84
+Yao-2017-1100-849	849	数据	statistics			4.73	6.27	4.55	5.05	3.82	4.08
+Yao-2017-1100-850	850	数量	quantity			4.41	3.68	4.45	4.45	7.42	6.87
+Yao-2017-1100-851	851	刷新	refresh			5.78	3.44	3.25	6.69	4.67	5.56
+Yao-2017-1100-852	852	衰老	age	AGE	1789	1.97	6.69	4.11	3.54	4.61	4.74
+Yao-2017-1100-853	853	衰弱	weak	WEAK	1601	6.71	5.37	5.17	3.59	5.65	5.02
+Yao-2017-1100-854	854	爽快	readily			7.06	6.08	7.83	7.67	7.63	7.53
+Yao-2017-1100-855	855	爽朗	bright and clear			7.27	4.33	6.83	6.86	6.76	6.76
+Yao-2017-1100-856	856	水稻	rice plant	WILD RICE	3390	5.69	4.31	4.75	5.08	4.95	5.62
+Yao-2017-1100-857	857	水晶	crystal			5.36	6.58	6.89	7.19	6.79	6.58
+Yao-2017-1100-858	858	水平	competence			4.44	3.39	4.63	5.31	4.81	4.7
+Yao-2017-1100-859	859	水仙	narcissus			3.41	5.23	6.5	5.8	5.29	4.49
+Yao-2017-1100-860	860	税收	tax	TAX	873	6.66	5.63	5.97	4.2	6.04	5.86
+Yao-2017-1100-861	861	睡眠	sleep	SLEEP (STATE)	2972	6.82	5.13	5.23	7.23	4.92	5.03
+Yao-2017-1100-862	862	顺序	order	ORDER	1128	4.22	4.34	5.63	5.1	6.89	6.28
+Yao-2017-1100-863	863	瞬间	instant			3.31	6.54	4.29	5.11	4.14	4.81
+Yao-2017-1100-864	864	司仪	host	HOST	1545	4.82	3.29	4.68	4.96	5.44	5.97
+Yao-2017-1100-865	865	私奔	elope			2.52	5.28	3.6	4.74	3.91	4.88
+Yao-2017-1100-866	866	思考	think	THINK (REFLECT)	1415	5.36	5.04	5.46	4.94	5.51	5.78
+Yao-2017-1100-867	867	思维	logic	LOGIC	2429	2.06	7.22	4.56	5	4.81	5.23
+Yao-2017-1100-868	868	死囚	Death row inmates			2.29	6.85	3.78	3.68	5.77	4.97
+Yao-2017-1100-869	869	死刑	death penalty			4.64	4.29	6.5	3.07	6.48	5.54
+Yao-2017-1100-870	870	死罪	death penalty			5.1	4.95	4.88	5.27	6.62	6.32
+Yao-2017-1100-871	871	寺院	temple	TEMPLES	1604	5.05	5.09	6.09	3.95	6.71	5.66
```

Comment:
Google pictures claims this is 911 TEMPLE.

Reply:
Oh yes, important spot, @CarolinHu, this is a Tempel.